src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -679,6 +682,7 @@
     <Compile Include="System\Runtime\InteropServices\StructLayoutAttribute.cs" />
     <Compile Include="System\WeakReference.cs" />
     <Compile Include="System\WeakReferenceOfT.cs" />
+    <Compile Include="..\..\Common\src\Internal\Text\UTF8String.cs" />
```

Comment:
Thanks. That seems better and simplifies the code a bit. I forgot non-windows is using UTF-8 as their "ANSI". Fixed.